R/pool.R

```diff
@@ -19,24 +18,33 @@ NULL
 #' @export
 Pool <- R6Class("Pool",
   public = list(
+
     valid = NULL,
     counters = NULL,
     minSize = NULL,
     maxSize = NULL,
     idleTimeout = NULL,
+    validateTimeout = NULL,
```

Comment:
Yeah. Or the 90/90 rule. The first 90% is easy, it's the second 90% that's hard... ;)